packages/clusview/metrics/__init__.py

```diff
@@ -1,3 +1,3 @@
 """
-Topic modelling performance metrics.
+Functions for collecting metrics on clusters such as average cluster size, count etc.
```

Comment:
I would remove mentioning specific metrics. I prefer leaving this to Intellisense when importing modules.